pkg/clients/dclient/discovery.go

```diff
@@ -17,8 +17,8 @@ import (
 // IDiscovery provides interface to mange Kind and GVR mapping
 type IDiscovery interface {
 	FindResource(groupVersion string, kind string) (apiResource, parentAPIResource *metav1.APIResource, gvr schema.GroupVersionResource, err error)
-	GetGVRFromKind(kind string) (schema.GroupVersionResource, error)
-	GetGVRFromAPIVersionKind(groupVersion string, kind string) schema.GroupVersionResource
+	// TODO: there's no mapping from GVK to GVR, this is very error prone
```

Comment:
I'm not clear how to solve this mapping issue between GVR and GVK. We need some ways to find the relationship.